### DISCUSSION TITLE
In the docs "media queries" is mentioned, but...

### DISCUSSION THREAD

**Discussion by smlarkin** [nativewind/nativewind#75] [General]
...I have missed any examples.

Do you support these: https://tailwindcss.com/docs/responsive-design ?

**Comment by marklawlor** [comment]
Yes they are supported! I'm currently working on the documentation including adding live examples :100:

**Reply by smlarkin** [reply]
Thanks for the reply, @marklawlor ...

A few other tangentially related questions ...

1) Is there a best practice for getting the most efficient/performant CSS on all platforms (web/mobile) if I need to dynamically style something that TW does not cover now?  Would I just mix-n-match between the `style` prop and the `className` prop ?  Should I use a `StyleSheet` ?

I know Nicolas Gallagher has tweaked RNW to for StyleSheet in v0.18 and unclear how to get the best of the best with tailwindcss-react-native for those times I need to reach outside of your TW support.

2) For Accessability, how will this work on both platforms.  TW supports these https://tailwindcss.com/docs/screen-readers , but unclear how they will apply to mobile.  Any ideas?

Cheers!

**Reply by marklawlor** [reply]
> Is there a best practice for getting the most efficient/performant CSS on all platforms (web/mobile) if I need to dynamically style something that TW does not cover now? Would I just mix-n-match between the style prop and the className prop ? Should I use a StyleSheet ?

You have two options, you can either use `StyleSheet.create` or you can write a custom Tailwind Plugin to add the features you need https://tailwindcss.com/docs/plugins

> I know Nicolas Gallagher has tweaked RNW to for StyleSheet in v0.18 and unclear how to get the best of the best with tailwindcss-react-native for those times I need to reach outside of your TW support.

If you enable `preview={true}` on the `<TailwindProvider>` it will take full advantage of RNW 0.18. This will be the default in the upcoming major release.

> For Accessability, how will this work on both platforms. TW supports these https://tailwindcss.com/docs/screen-readers , but unclear how they will apply to mobile. Any ideas?

Accessibility on native & web is very different and not a problem I have tackled yet. For the mean time I would recommend using both the web accessibility classes and native accessibility props.

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
